src/toil/batchSystems/kubernetes.py

```diff
@@ -803,6 +809,26 @@ def _check_accelerator_request(self, requirer: Requirer) -> None:
                     ],
                 )
 
+    class FakeResponse:
+        data: str
+
+    T = TypeVar('T')
+    def _load_kubernetes_object(self, file: str, cls: type[T]) -> T:
+        """
+        Deserialize a YAML representation into a Kubernetes object
+        :param file: Path to YAML file
+        :param cls: Kubernetes API model type for deserialized object
+        :return: Deserialized object
+        """
+        YAML = yaml.YAML(typ='safe')
+        object_def = YAML.load(open('container.yaml').read())
```

Comment:
Reading and parsing the file for every pod might not be super efficient; we *could* read the files and make the objects once at startup and just re-use them, since we never mutate them except for setting the privileged flag, which we could also do at startup. I'm not sure how much faster that would be, or whether Kubernetes lets us share parts of the object graph like that.